cxx-qt-gen/src/writer/rust/mod.rs

```diff
@@ -18,9 +66,18 @@ pub fn write_rust(generated: &GeneratedRustBlocks) -> TokenStream {
     let cxx_qt_mod_ident = format_ident!("cxx_qt_{}", cxx_mod_ident);
 
     // Retrieve the module contents and namespace
-    let cxx_mod = &generated.cxx_mod;
+    let mut cxx_mod = generated.cxx_mod.clone();
     let cxx_qt_mod_contents = &generated.cxx_qt_mod_contents;
     let namespace = &generated.namespace;
+    let namespace_internals = &generated.namespace_internals;
+
+    // Inject the common methods into the bridge which we need
+    write_cxx_common_methods(
```

Comment:
I feel like this shouldn't be handled by passing a mutable reference to the cxx_mod.
It leads to code duplication in terms of `content.push(syn::parse2(...)).expect("...")`.

Maybe refactor write_cxx_common_methods to:
`cxx_common_blocks` that returns a `Vec<TokenStream>`.
Then just loop over that:
``` rust
let cxx_mod_items = cxx_mod.content.as_mut().expect("").1;
for block in cxx_common_blocks(cpp_struct_ident, rust_struct_ident, namespace_internals){ 
    cxx_mod_items.push(syn::parse2(block).expect("Could not build extern block!"));
}
```

Reply:
Done